keps/sig-api-machinery/3488-cel-admission-control/README.md

```diff
@@ -2387,15 +2471,9 @@ well as the [existing list] of feature gates.
 [existing list]: https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/
 -->
 
-- [ ] Feature gate (also fill in values in `kep.yaml`)
-  - Feature gate name: CELValidatingAdmission
+- [X] Feature gate (also fill in values in `kep.yaml`)
+  - Feature gate name: ValidatingAdmissionPolicy
```

Comment:
Did we change the feature gate name or was it just wrong in here before?

Reply:
Good question. It was wrong in the KEP but is correct in the code: https://github.com/kubernetes/kubernetes/blob/c840c947554e10e6b2918e731c86dbd0e541361a/staging/src/k8s.io/apiserver/pkg/features/kube_features.go#L105

Reply:
Just FYI: we decided to change the feature gate name while implementing the feature since it better represent the feature, here just as KEP update to reflect what we'v used in code. :)